### PR TITLE
Insights management tweaks

### LINKED ISSUE #10974: Stats Insights Management tweaks
We'd like to make a few changes to Insights Management to better align it with iOS and the way users are using the feature:

- [x] Replace the Edit button at the bottom with an "Add stats card" row – here's how it should look:

C6B0F8DD-1D41-4EEC-84DC-0453F8F7E001

- [x] Update the Manage Insights screen to list all card types grouped by section, where cards can be tapped to add them to the Insights screen. Remove reordering and removing card functionality from this screen:

Screenshot 2019-12-17 at 13 30 36

- [x] Update the default set of cards to the following:

* Latest Post Summary
* Today
* All-Time
* Follower Totals

### PR DESCRIPTION
**Pull request by planarvoid**
Fixes #10974

This PR contains 3 changes. 
- It replaces the "Edit" button we used to access Insight management with "Add new stats card" button. 
- It updates the insights management screen design based on the issue
- The default insights are now changed (in the linked FluxC PR - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1460)

Screenshot 2019-12-20 at 10 33 38

Screenshot_1576834423

To test:
- Open Stats/Insights
- Scroll down
- Check that the "Add new stats card" is visible and has the updated design
- Click on it
- Check that you see insights with the updated design
- Try to add/remove a few insight types
- Click on Save
- Check that the cards are added/removed/updated
- Click on the "Add new stats card" again
- Try to add/remove a few insight types
- Cancel the changes without saving
- The cards are not added/removed/updated

PR submission checklist:

- [x] I have considered adding unit tests where possible.

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by jkmassel** [comment]
I'm moving this PR to 14.1 since we're code freezing today.

**Comment by jkmassel** [comment]
We're freezing `14.2` today, so this PR is being bumped to `14.3`. If you'd like it to ship with `14.2`, please merge it into `release/14.2` and ping me – I'll be happy to cut a new beta release.

**Comment by oguzkocer** [comment]
@develric It looks like this PR has been opened 2 months ago, but hasn't been reviewed yet. Do you mind having a look at it? Thanks!

**Comment by develric** [comment]
Hi @oguzkocer 👋! It looks like I was missing it so thanks for the ping 🙇‍♂️! I'm not sure I will be able to look into it today but should be able to take care next week (I will prioritize this one). Hope it's fine.

**Comment by oguzkocer** [comment]
We're freezing `14.3` today, so this PR is being bumped to `14.4`. If you'd like it to ship with `14.3`, please merge it into `release/14.3` and ping me – I'll be happy to cut a new beta release.

**Review by develric — APPROVED** [review]
Hi @planarvoid 😊! Checked this PR and it's looking good. Also tested according to testing steps and didn't find any issue! 

I tagged the [companion FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1460) as 1.6.4-beta1 and linked it in the project build.gradle .

I think what I did should be enough/correct but since it's my first fluxC tagging I would ping @malinajirka or @oguzkocer (hoping one of you could have a look) just to confirm that I did not miss anything in the tagging/link process. A part from this I'm approving this PR and I'm fine to merge after this last quick check 👍!

File: build.gradle [hunk 1]
```diff
@@ -96,7 +96,7 @@ buildScan {
 
 ext {
     daggerVersion = '2.22.1'
-    fluxCVersion = '1.6.4'
+    fluxCVersion = '1.6.4-beta1'
```
Comment (oguzkocer) [inline build.gradle]: @develric @malinajirka mentioned that the beta tags start with the next version number, so it should be `1.6.5`. Also, there is a `-` between `beta` and the minor number.

I've deleted `1.6.4-beta1` tag and created `1.6.5-beta-1`, so you can just accept the following change and it should work.

```suggestion
    fluxCVersion = '1.6.5-beta-1'
```

**Comment by develric** [comment]
@oguzkocer , @malinajirka thanks so much for the clarification 🙇‍♂️. Accepted the suggested modification and going to merge now 👍!

**Pull request merged**